### ISSUE TITLE
mails_from preference not defined in 3-0 stable

### ISSUE THREAD

**Issue by kurapatijayaram** [spree-contrib/spree_mail_settings#9]
Hi ..... I installed 3-0 stable version of spree_mail_settings I am getting the following error :

mails_from preference not defined
Extracted source (around line #62):
         

  def has_preference!(name)
    raise NoMethodError.new "#{name} preference not defined" unless has_preference? name
  end

  def has_preference?(name)

pls help me ...
thank u

**Comment by kurapatijayaram** [comment]
It is expecting mails_from preference , If I add that particular preference in the model means its working ....

But mail is not delivering ....... I am able to see the mail template in the log ...

**Comment by Mrjaco12** [comment]
I'm experiencing the same issue. I was able to resolve it by adding the following to an initializer:
```ruby
Spree::AppConfiguration.class_eval do
  preference :mails_from, :string
end
```
I feel like this should be defined in the gem somewhere?

**Comment by kurapatijayaram** [comment]
Hi @Mrjaco12  ... ya, even I achieved by adding that preference. There is no need to define somewhere in the gem , we can define in app_configuaration model (app_configuaration_decorator.rb) which consists of all preferences.....

Thank you

**Comment by futhr** [comment]
Fixed with #12
